airbyte-integrations/connectors/source-webflow/unit_tests/test_source.py

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from unittest import TestCase
+from unittest.mock import MagicMock
+
+from source_webflow.source import Info, SourceWebflow
+
+
+def test_check_connection(mocker):
+    source = SourceWebflow()
+    fake_info_record = {"info": "is_mocked"}
```

Comment:
Hi @alexander-marquardt, correct me if I'm wrong here: with the latest changes to source.py and the check_connection function, the test_check_connection unit test is now stale and the build is failing. Would it be possible for you to fix this?

Reply:
You are correct - I broke the functionality. Should be fixed now.